src/components/container/FeaturedJobs/FeaturedJobs.js

```diff
@@ -0,0 +1,34 @@
+import { staticFeaturedJobs } from '../../../state/models/staticFeaturedJobs';
+import viewMoreIcon from './assets/icons/viewMore.svg';
+import amazonLogo from './assets/logos/amazon.png';
+import quantumLogo from './assets/logos/quantum.png';
+import './FeaturedJobs.scss';
+
+
+function FeaturedJobs() {
+    const logoNameToPath = {
+        'Amazon': amazonLogo,
+        'Quantum': quantumLogo,
+    }
+
+    return (
+        <section className="FeaturedJobs">
```

Comment:
I love this! But one last improvement:

In BEM-classes only the first element should be in PascalCase, so for example

instead of `FeaturedJobs__ViewMoreLink__Text` it should be `FeaturedJobs__viewMoreLink__text`

Reply:
Thanks for the hint, I did not know that :)